src/js/rtc_session.js

```diff
@@ -477,13 +477,82 @@ export default class RtcSession {
     get sessionReport() {
         return this._sessionReport;
     }
-
     get callId() {
         return this._callId;
     }
     get mediaStream() {
         return this._userAudioStream;
     }
+    get localVideoStream() {
+        return this._streamToBeClosed;
+    }
+    get remoteVideoStream() {
+        return this._remoteVideoStream;
+    }
+    pauseLocalVideo() {
+        if (this._streamToBeClosed) {
+            var videoTrack = this._streamToBeClosed.getTracks()[1];
+            if(videoTrack) {
+                videoTrack.enabled = false;
+            }
+        }
+    }
+    resumeLocalVideo() {
+        if (this._streamToBeClosed) {
+            var videoTrack = this._streamToBeClosed.getTracks()[1];
+            if(videoTrack) {
+                videoTrack.enabled = true;
+            }
+        }
+    }
+    pauseRemoteVideo() {
+        if (this._remoteVideoStream) {
+            var videoTrack = this._remoteVideoStream.getTracks()[1];
+            if(videoTrack) {
+                videoTrack.enabled = false;
+            }
+        }
+    }
+    resumeRemoteVideo() {
+        if (this._remoteVideoStream) {
+            var videoTrack = this._remoteVideoStream.getTracks()[1];
+            if(videoTrack) {
+                videoTrack.enabled = true;
+            }
+        }
+    }
+    pauseLocalAudio() {
+        if (this._streamToBeClosed) {
+            var audioTrack = this._streamToBeClosed.getTracks()[0];
```

Comment:
fixed. In the most-recent commit, use getAudioTracks() & getVideoTracks() instead of getTracks(), so we don't have to worry about the order of audio/video tracks.